lighthouse-core/test/report/html/renderer/psi-test.js

```diff
@@ -20,10 +20,16 @@ const CriticalRequestChainRenderer =
     require('../../../../report/html/renderer/crc-details-renderer.js');
 
 const sampleResultsStr = fs.readFileSync(__dirname + '/../../../results/sample_v2.json', 'utf-8');
-const sampleResultsRoundtripStr = fs.readFileSync(
-  __dirname + '/../../../../../proto/sample_v2_round_trip.json',
-  'utf-8'
-);
+
+let sampleResultsRoundtripStr;
+try {
+  sampleResultsRoundtripStr =
+    fs.readFileSync(__dirname + '/../../../results/sample_v2.json', 'utf-8');
+} catch (err) {
+  // eslint-disable-next-line no-console
+  console.warn('Skipping test - you need to run yarn test-proto first.');
```

Comment:
The `details-renderer-test.js` console errors currently emitted during `yarn unit-core` are already distracting (visually triggering as if there was a failure), can we avoid adding more of them? Is a comment in the file sufficient? (since you'd presumably come here to diagnose the problem)

Reply:
> The `details-renderer-test.js` console errors currently emitted during `yarn unit-core` are already distracting (visually triggering as if there was a failure), can we avoid adding more of them? Is a comment in the file sufficient? (since you'd presumably come here to diagnose the problem)

oh, do `console.warn` messages get skipped in jest output? I don't see these in the travis log

Reply:
Nah, but I was reading the wrong file here, perhaps that's why you didn't see it.

![image](https://user-images.githubusercontent.com/4071474/79395911-8cc2f980-7f2f-11ea-9d25-cbb81a31fac3.png)